imagery/i.ortho.photo/lib/find_init.c

```diff
@@ -4,15 +4,19 @@
  * Find the a camera initial file in the current group (if it exists)
  **************************************************************/
 #include <grass/gis.h>
+#include <grass/glocale.h>
 
 int I_find_initial(char *group)
 {
-    char *element;
-
-    element = (char *)G_malloc(80 * sizeof(char));
+    char element[GNAME_MAX + 6];
 
     if (group == NULL || *group == 0)
         return 0;
-    sprintf(element, "group/%s", group);
+
+    if (snprintf(element, GNAME_MAX, "group/%s", group) >= GNAME_MAX) {
+        G_warning(_("Group name <%s> truncated"), group);
```

Comment:
“Group name <%s> truncated” sounds to me: “the group name was just shortened a bit, but we’re fine…let’s continue…”. Perhaps better to note it was too long.

Reply:
yes changed the message